pkg/source/url_entry.go

```diff
@@ -0,0 +1,44 @@
+package source
```

Comment:
Remove this package and replace it with struct.

Reply:
removed